docs/api/params-for-successful-requests.md

```diff
@@ -0,0 +1,50 @@
+# Parameters for successful requests
+
+This section highlights the required and optional parameters for the To-Do service.
+
+To get a successful response, at a minimum, you must specify the properties listed under each task. 
+
+# Add a new task
```

Comment:
In a markdown file, there can only be one H1 title, which is the topic title.

Reply:
Got it! Will update..